exporters/common/src/main/java/io/opentelemetry/exporter/internal/marshal/Marshaler.java

```diff
@@ -32,7 +32,7 @@ public final void writeJsonTo(OutputStream output) throws IOException {
   }
 
   /** Marshals into the {@link JsonGenerator} in proto JSON format. */
-  public final void writeJsonTo(JsonGenerator output) throws IOException {
+  public final void writeJsonToGenerator(JsonGenerator output) throws IOException {
```

Comment:
maybe worth a comment so we don't rename it back? (which seems like something I would do)

```suggestion
  // intentionally not overloading writeJsonTo(OutputStream) in order to avoid compilation dependency
  // on jackson when using writeJsonTo(OutputStream)
  // (see https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1551#discussion_r1849064365)
  public final void writeJsonToGenerator(JsonGenerator output) throws IOException {
```